RustFxA/FxAWebViewModel.swift

```diff
@@ -0,0 +1,330 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
nit: requires extra spacing next to * comment

```
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Thanks @LamourBt for the changes. 

I am going to approve the PR although I found that the FxAWebViewModel doesn't have proper header (lint issue). 

If you have some time to update the the header then that would be amazing otherwise I'll just add another commit after merging this.

Reply:
Hi @nbhasin2, I copied and pasted your header but git hasn't found any changes :(

Reply:
@LamourBt Its our the header formatting issue. This also confuses me at times when I copy header from one file to another it doesn't copy spaces 🤦‍♂️ 

Basically in your file add one space starting on line 2 and 3 and see if that works. 

If it doesn't work I still plan on merging your change tomorrow after doing some local testing on my device :)

Reply:
Thanks again @LamourBt for all the contributions you make 😊